### ISSUE TITLE
Playing network stream with parameter

### ISSUE THREAD

**Issue by Manu1985** [higankanshi/xZune.Vlc#36]
Hello,

I am using xZune and I am very happy with it. In VLC Player I can set parameter if I want to Play a Network stream. The parameter can be for example:

:avformat-format=mxg

It is possible to set this parameter in xZune if I want to Play a Network stream?

Thanks.

**Comment by higankanshi** [comment]
Try
```CSHARP
LoadMediaWithOptions(Uri uri, String options);

AddOption(String option);
```

**Comment by Manu1985** [comment]
LoadMediaWithOptions() is working. Thanks for the fast answer.

Now I have to get the .net 3.5 sample working. I got some Errors if I try to compile it.

**Comment by higankanshi** [comment]
I know it...I changed some code to fix bug #32 ...
These code maybe have some wrong with .Net 3.5...
I'll fix it later.

**Comment by higankanshi** [comment]
I fix all wrong in  .Net 3.5 ...
Now you can compile xZune.Vlc for .Net 3.5 ...
